src/Files.App/Helpers/AdaptiveLayoutHelpers.cs

```diff
@@ -92,10 +92,10 @@ public static bool PredictLayoutMode(FolderSettingsViewModel folderSettings, str
 				foldersCount = filesAndFolders.Where((item) => item.PrimaryItemAttribute == StorageItemTypes.Folder).Count();
 				miscFilesCount = allItemsCount - (mediaCount + imagesCount + foldersCount);
 
-				mediaPercentage = (float)((float)mediaCount / (float)allItemsCount) * 100.0f;
-				imagesPercentage = (float)((float)imagesCount / (float)allItemsCount) * 100.0f;
-				foldersPercentage = (float)((float)foldersCount / (float)allItemsCount) * 100.0f;
-				miscFilesPercentage = (float)((float)miscFilesCount / (float)allItemsCount) * 100.0f;
+				mediaPercentage = (float)(mediaCount / (float)allItemsCount) * 100.0f;
+				imagesPercentage = (float)(imagesCount / (float)allItemsCount) * 100.0f;
+				foldersPercentage = (float)(foldersCount / (float)allItemsCount) * 100.0f;
+				miscFilesPercentage = (float)(miscFilesCount / (float)allItemsCount) * 100.0f;
```

Comment:
If you change this to `mediaCount * 100.0f / allItemsCount` you can remove all the casts

Reply:
Done